tests/conftest.py

```diff
@@ -1,10 +1,14 @@
-import os
-import sys
+import logging
 
 import pytest
 
 from app.main import app
 
+asyncio_logger = logging.getLogger("asyncio")
+asyncio_logger.setLevel(logging.INFO)
```

Comment:
Can you add mini comments here? I get that they are logging lol. But I don't know anything other than that. Maybe there is nothing else to add.

Reply:
The [blame](https://github.com/elthran/UDK/blame/8c2ff717d3796c3ffed1b01b3e419048e24c262b/tests/conftest.py) for this line of code should say why ... but I can put the comment in-line.